src/Runner.Worker/ExecutionContext.cs

```diff
@@ -335,7 +336,7 @@ public TaskResult Complete(TaskResult? result = null, string currentOperation =
             }
 
             // report total delay caused by server throttling.
-            if (_totalThrottlingDelayInMilliseconds > 0)
+            if (_totalThrottlingDelayInMilliseconds > _throttlingDelayReportThreshold)
```

Comment:
Discussed offline, this is related to log lines and timeline record updates. Not general perf tarpitting.

Upload artifact, etc would be reported as 429 during that experience (depends on whether they log).

Reply:
Existing code so better to just leave it, consider Timespan in future so name doesn't need to convey units :)